config/default.yaml

```diff
@@ -185,10 +185,6 @@ protections:
     #
     # WordList will ban users who use these words when first joining a room, so take caution when selecting them.
     #
-    # For advanced usage, regex can also be used, see the following links for more information;
-    #  - https://www.digitalocean.com/community/tutorials/an-introduction-to-regular-expressions
-    #  - https://regexr.com/
-    #  - https://regexone.com/
```

Comment:
We should probably explicitly say that they don't support regexs

Reply:
meow thats a sensible change but im not sure how you would like it to be worded.

Reply:
The word list protection does not support regular expressions at this time.

Reply:
Wording added.